iothub/device/samples/PnpDeviceSamples/TemperatureController/ComponentTemperatureControllerSampleNew.cs

```diff
@@ -95,18 +95,13 @@ public async Task PerformOperationsAsync(CancellationToken cancellationToken)
                 HumidityRange targetHumidityRange = humidityRangeRequested.ToObject<HumidityRange>();
                 _logger.LogDebug($"Property: Received - component=\"{Thermostat1}\", {{ \"{propertyName}\": {targetHumidityRange} }}.");
 
-                var temperatureUpdateResponse = new NewtonsoftJsonWritablePropertyResponse(
-                    targetHumidityRange,
-                    (int)StatusCode.Completed,
-                    writableProperties.Version,
-                    "The operation completed successfully.");
-
                 var propertyPatch = new ClientPropertyCollection
                 {
-                    { propertyName, temperatureUpdateResponse, Thermostat1 }
+                    { propertyName, "1", Thermostat1 },
```

Comment:
nit: just for this sample we should modify the property name and component name here -> else it'll be overwritten on the final patch

Reply:
yeah was testing out that accessor, it should be back to normal.